src/4cade.init.a

```diff
@@ -408,6 +408,12 @@ CopyDevs
          bcc   @outer
 
          bit   CLEARKBD
+
+         +READ_ROM_NO_WRITE
```

Comment:
this code needs to be in `Reenter` so it runs when we return from a demo or a game

Reply:
Not sure I understand. Reenter would happen with LC bank1 switched in, right?
I do two ROM calls then switch back to bank 1. Doesn't @start happen after each demo or slideshow group? We actually don't want to hit TEXT after everything because it causes graphics glitches.